src/test/java/com/google/cloud/spanner/jdbc/it/ITJdbcSimpleStatementsTest.java

```diff
@@ -188,15 +182,16 @@ public void testBatchedDdlStatements() throws SQLException {
   }
 
   @Test
-  public void testAddBatchWhenAlreadyInBatch() {
+  public void testAddBatchWhenAlreadyInBatch() throws SQLException {
     try (Connection connection = createConnection(env, database)) {
-      connection.createStatement().execute("START BATCH DML");
-      connection.createStatement().addBatch("INSERT INTO Singers (SingerId) VALUES (-1)");
-      fail("missing expected exception");
```

Comment:
This limitation has been lifted in the Java Connection API, which means that we can now allow this in the JDBC driver as well. Executing a DML batch when a batch is already active, is supported and just adds these batched statements to the active batch.